StackExchange.Redis/StackExchange/Redis/GeoEntry.cs

```diff
@@ -0,0 +1,284 @@
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Remoting.Metadata.W3cXsd2001;
+
+namespace StackExchange.Redis
+{
+    /// <summary>
+    /// GeoRadius command options.
+    /// </summary>
+    [Flags]
+    public enum GeoRadiusOptions
+    {
+        /// <summary>
+        /// No Options
+        /// </summary>
+        None = 0,
+        /// <summary>
+        /// Redis will return the coordinates of any results.
+        /// </summary>
+        WITHCOORD = 1,
+        /// <summary>
+        /// Redis will return the distance from center for all results.
+        /// </summary>
+        WITHDIST = 2,
+        /// <summary>
+        /// Redis will return the geo hash value as an integer. (This is the score in the sorted set)
+        /// </summary>
+        WITHHASH = 4
+    }
+
+    /// <summary>
+    /// The result of a GeoRadius command.
+    /// </summary>
+    public struct GeoRadiusResult
+    {
+        internal readonly RedisValue member;
+        internal readonly GeoRadius geoRadius;
+        internal readonly double? distanceFromCenter;
+        internal readonly long? geoHash;
+        internal readonly GeoPosition? geoPosition;
+
+        /// <summary>
+        /// The matched member.
+        /// </summary>
+        public RedisValue Member => member;
```

Comment:
I wonder if all of these should just be `{get;}` (and remove the fields); note that I haven't read end-to-end yet, so I don't have much context of what this type is doing yet